lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -517,6 +517,17 @@ public function prepare_item_for_response( $user, $request ) {
 		}
 
 		$context = ! empty( $request['context'] ) ? $request['context'] : 'embed';
+
+		if ( 'edit' === $context ) {
+			if ( empty( $data['capabilities'] ) ) {
+				$data['capabilities'] = new stdClass();
```

Comment:
> Can you take a look to see if there's prior art?

I took a look at the codebase. For the date fields, we use `null` instead of an empty string if there's no date stored. For `parent`, we use `null` if `0` is stored in the database. I think we should probably use `null` instead of an empty object.